_episodes/05-function-style.md

```diff
@@ -72,34 +89,48 @@ canvas(with_attribution=True)
 ~~~
 {: .output}
 
-If we try the same thing on our `mean` function, we don't get a helpful error message.
+If we try the same thing on our `open_pdb` function, we don't get a helpful error message.
 
-We will want to write a docstring for our new mean function. This way, it will be clear to new developers who use our code what the function does, and be accessible to any users using the function interactively. Returning, to the `molssi_math.py` module file, edit your `mean` function to look like the following:
+We will want to write a docstring for our new mean function. This way, it will be clear to new developers who use our code what the function does, and be accessible to any users using the function interactively. Returning, to the `pdb.py` module file, edit your `open_pdb` function to look like the following:
 
 ~~~
-def mean(num_list):
-    """
-    Calculate the mean/average of a list of numbers.
+def open_pdb(file_location):
+    """Open and read coordinates and atom symbols from a pdb file.
+
+    The pdb file must specify the atom elements in the last column, and follow
+    the conventions outlined in the PDB format specification.
 
     Parameters
     ----------
-    num_list : list
-        The list to take the average of
+    file_location : str
+        The location of the xyz file to read in.
```

Comment:
This was probably also an error I the original code, but it should say "The location of the pdb file to read in" (instead of xyz file).